From e1539d442b4088e5afed793e87abb2b72c7a29d5 Mon Sep 17 00:00:00 2001
From: Alyssa Ketpreechasawat <kalyssa@google.com>
Date: Tue, 12 Mar 2024 07:57:28 +0000
Subject: [PATCH] DO NOT MERGE: Drop prebuilt files to android source tree.

The following files have been updated as part of this change:

current/Android.bp
current/snapshot-creation-build-number.txt
current/java_systemserver_libs/snapshot/jars/are/invalid/service-wifi.jar
current/licenses/build/soong/licenses/LICENSE
current/hiddenapi/annotation-flags.csv
current/hiddenapi/filtered-flags.csv
current/hiddenapi/filtered-stub-flags.csv
current/hiddenapi/hiddenapi-max-target-o-low-priority.txt
current/hiddenapi/hiddenapi-max-target-r-low-priority.txt
current/hiddenapi/hiddenapi-unsupported.txt
current/hiddenapi/index.csv
current/hiddenapi/metadata.csv
current/hiddenapi/signature-patterns.csv
current/sdk_library/system/framework-wifi-removed.txt
current/sdk_library/system/framework-wifi-stubs.jar
current/sdk_library/system/framework-wifi.srcjar
current/sdk_library/system/framework-wifi.txt
current/sdk_library/system/framework-wifi_annotations.zip
current/sdk_library/module-lib/framework-wifi-removed.txt
current/sdk_library/module-lib/framework-wifi-stubs.jar
current/sdk_library/module-lib/framework-wifi.srcjar
current/sdk_library/module-lib/framework-wifi.txt
current/sdk_library/module-lib/framework-wifi_annotations.zip
current/sdk_library/public/framework-wifi-removed.txt
current/sdk_library/public/framework-wifi-stubs.jar
current/sdk_library/public/framework-wifi.srcjar
current/sdk_library/public/framework-wifi.txt
current/sdk_library/public/framework-wifi_annotations.zip
current/compat_configs/wifi-compat-config/wifi-compat-config_meta.xml

Built here: ab/T1006561

This build IS suitable for public release.

The change is generated with prebuilt drop tool.

Bug: 267231571
Test: presubmit
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:2ff2726e38819c05769e09dee3e0dfe56ac7e857)
Merged-In: I1c154a50175ee494da321ff952c5edf9313d0d07
Change-Id: I1c154a50175ee494da321ff952c5edf9313d0d07
---
 .../prebuilt_info_current_current_zip.asciipb |  2 +-
 current/hiddenapi/filtered-flags.csv          | 38 ++++--------
 current/hiddenapi/filtered-stub-flags.csv     | 20 +++----
 current/hiddenapi/index.csv                   | 58 +++++++++----------
 current/hiddenapi/metadata.csv                |  3 +
 current/hiddenapi/signature-patterns.csv      |  1 +
 current/snapshot-creation-build-number.txt    |  2 +-
 7 files changed, 55 insertions(+), 69 deletions(-)

diff --git a/.prebuilt_info/prebuilt_info_current_current_zip.asciipb b/.prebuilt_info/prebuilt_info_current_current_zip.asciipb
index a6421fe..bb54777 100644
--- a/.prebuilt_info/prebuilt_info_current_current_zip.asciipb
+++ b/.prebuilt_info/prebuilt_info_current_current_zip.asciipb
@@ -1,6 +1,6 @@
 drops {
   android_build_drop {
-    build_id: "T1005166"
+    build_id: "T1006561"
     target: "train_build"
     source_file: "mainline-sdks/for-Tiramisu-build/current/com.google.android.wifi/sdk/wifi-module-sdk-current.zip"
   }
diff --git a/current/hiddenapi/filtered-flags.csv b/current/hiddenapi/filtered-flags.csv
index 24446a3..d31b510 100644
--- a/current/hiddenapi/filtered-flags.csv
+++ b/current/hiddenapi/filtered-flags.csv
@@ -356,16 +356,17 @@ Landroid/net/wifi/IWifiScanner$Default;->asBinder()Landroid/os/IBinder;,public-a
 Landroid/net/wifi/IWifiScanner$Stub$Proxy;-><init>(Landroid/os/IBinder;)V,unsupported
 Landroid/net/wifi/IWifiScanner$Stub$Proxy;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/IWifiScanner$Stub$Proxy;->getInterfaceDescriptor()Ljava/lang/String;,lo-prio,max-target-o
-Landroid/net/wifi/IWifiScanner$Stub$Proxy;->getMessenger()Landroid/os/Messenger;,lo-prio,max-target-o
 Landroid/net/wifi/IWifiScanner$Stub$Proxy;->mRemote:Landroid/os/IBinder;,unsupported
 Landroid/net/wifi/IWifiScanner$Stub;-><init>()V,unsupported
 Landroid/net/wifi/IWifiScanner$Stub;->DESCRIPTOR:Ljava/lang/String;,lo-prio,max-target-o
 Landroid/net/wifi/IWifiScanner$Stub;->TRANSACTION_getAvailableChannels:I,lo-prio,max-target-o
-Landroid/net/wifi/IWifiScanner$Stub;->TRANSACTION_getMessenger:I,lo-prio,max-target-o
 Landroid/net/wifi/IWifiScanner$Stub;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/IWifiScanner$Stub;->asInterface(Landroid/os/IBinder;)Landroid/net/wifi/IWifiScanner;,unsupported
 Landroid/net/wifi/IWifiScanner$Stub;->onTransact(ILandroid/os/Parcel;Landroid/os/Parcel;I)Z,public-api,sdk,system-api,test-api
-Landroid/net/wifi/IWifiScanner;->getMessenger()Landroid/os/Messenger;,lo-prio,max-target-o
+Landroid/net/wifi/IWifiScannerListener$Default;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Stub$Proxy;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Stub;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Stub;->onTransact(ILandroid/os/Parcel;Landroid/os/Parcel;I)Z,public-api,sdk,system-api,test-api
 Landroid/net/wifi/IWifiVerboseLoggingStatusChangedListener$Default;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/IWifiVerboseLoggingStatusChangedListener$Stub$Proxy;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/IWifiVerboseLoggingStatusChangedListener$Stub;->asBinder()Landroid/os/IBinder;,public-api,sdk,system-api,test-api
@@ -2235,6 +2236,7 @@ Landroid/net/wifi/WifiNetworkSuggestion;->isUntrusted()Z,public-api,sdk,system-a
 Landroid/net/wifi/WifiNetworkSuggestion;->isUserInteractionRequired()Z,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiNetworkSuggestion;->toString()Ljava/lang/String;,core-platform-api,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiNetworkSuggestion;->writeToParcel(Landroid/os/Parcel;I)V,public-api,sdk,system-api,test-api
+Landroid/net/wifi/WifiScanner$$ExternalSyntheticLambda0;->run()V,core-platform-api,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ActionListener;->onFailure(ILjava/lang/String;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ActionListener;->onSuccess()V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$BssidInfo;-><init>()V,sdk,system-api,test-api
@@ -2256,14 +2258,6 @@ Landroid/net/wifi/WifiScanner$HotlistSettings;->apLostThreshold:I,sdk,system-api
 Landroid/net/wifi/WifiScanner$HotlistSettings;->bssidInfos:[Landroid/net/wifi/WifiScanner$BssidInfo;,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$HotlistSettings;->describeContents()I,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$HotlistSettings;->writeToParcel(Landroid/os/Parcel;I)V,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult$1;->newArray(I)[Ljava/lang/Object;,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult;-><init>(ILjava/lang/String;)V,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner$OperationResult;->CREATOR:Landroid/os/Parcelable$Creator;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner$OperationResult;->describeContents()I,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult;->description:Ljava/lang/String;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner$OperationResult;->reason:I,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner$OperationResult;->writeToParcel(Landroid/os/Parcel;I)V,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ParcelableScanData$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ParcelableScanData$1;->newArray(I)[Ljava/lang/Object;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ParcelableScanData;-><init>([Landroid/net/wifi/WifiScanner$ScanData;)V,sdk,system-api,test-api
@@ -2349,12 +2343,11 @@ Landroid/net/wifi/WifiScanner$ScanSettings;->setRnrSetting(I)V,sdk,system-api,te
 Landroid/net/wifi/WifiScanner$ScanSettings;->stepCount:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ScanSettings;->type:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$ScanSettings;->writeToParcel(Landroid/os/Parcel;I)V,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda0;->run()V,core-platform-api,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda1;->run()V,core-platform-api,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda2;->run()V,core-platform-api,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda3;->run()V,core-platform-api,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda4;->run()V,core-platform-api,public-api,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler;->handleMessage(Landroid/os/Message;)V,public-api,sdk,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda0;->run()V,core-platform-api,public-api,sdk,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda1;->run()V,core-platform-api,public-api,sdk,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda2;->run()V,core-platform-api,public-api,sdk,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda3;->run()V,core-platform-api,public-api,sdk,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda4;->run()V,core-platform-api,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$WifiChangeListener;->onChanging([Landroid/net/wifi/ScanResult;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$WifiChangeListener;->onQuiescence([Landroid/net/wifi/ScanResult;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner$WifiChangeSettings$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,sdk,system-api,test-api
@@ -2389,7 +2382,6 @@ Landroid/net/wifi/WifiScanner;->CMD_STOP_PNO_SCAN:I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->CMD_STOP_SINGLE_SCAN:I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->DBG:Z,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->GET_AVAILABLE_CHANNELS_EXTRA:Ljava/lang/String;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner;->INVALID_KEY:I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->MAX_SCAN_PERIOD_MS:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->MIN_SCAN_PERIOD_MS:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->PNO_PARAMS_PNO_SETTINGS_KEY:Ljava/lang/String;,lo-prio,max-target-o
@@ -2424,29 +2416,20 @@ Landroid/net/wifi/WifiScanner;->WIFI_BAND_UNSPECIFIED:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->WIFI_RNR_ENABLED:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->WIFI_RNR_ENABLED_IF_WIFI_BAND_6_GHZ_SCANNED:I,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->WIFI_RNR_NOT_NEEDED:I,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner;->addListener(Landroid/net/wifi/WifiScanner$ActionListener;)I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->configureWifiChange(IIIII[Landroid/net/wifi/WifiScanner$BssidInfo;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->configureWifiChange(Landroid/net/wifi/WifiScanner$WifiChangeSettings;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->getAvailableChannels(I)Ljava/util/List;,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner;->getListenerKey(Ljava/lang/Object;)I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->getScanResults()Z,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->getSingleScanResults()Ljava/util/List;,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->isScanning()Z,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->mContext:Landroid/content/Context;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner;->mInternalHandler:Landroid/os/Handler;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner;->mListenerKey:I,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner;->mListenerMap:Landroid/util/SparseArray;,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->mListenerMapLock:Ljava/lang/Object;,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->mService:Landroid/net/wifi/IWifiScanner;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner;->putListener(Ljava/lang/Object;)I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->registerScanListener(Landroid/net/wifi/WifiScanner$ScanListener;)V,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->registerScanListener(Ljava/util/concurrent/Executor;Landroid/net/wifi/WifiScanner$ScanListener;)V,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner;->removeListener(I)Ljava/lang/Object;,lo-prio,max-target-o
-Landroid/net/wifi/WifiScanner;->removeListener(Ljava/lang/Object;)I,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->setScanningEnabled(Z)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->startBackgroundScan(Landroid/net/wifi/WifiScanner$ScanSettings;Landroid/net/wifi/WifiScanner$ScanListener;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->startBackgroundScan(Landroid/net/wifi/WifiScanner$ScanSettings;Landroid/net/wifi/WifiScanner$ScanListener;Landroid/os/WorkSource;)V,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner;->startPnoScan(Landroid/net/wifi/WifiScanner$ScanSettings;Landroid/net/wifi/WifiScanner$PnoSettings;I)V,lo-prio,max-target-o
 Landroid/net/wifi/WifiScanner;->startScan(Landroid/net/wifi/WifiScanner$ScanSettings;Landroid/net/wifi/WifiScanner$ScanListener;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->startScan(Landroid/net/wifi/WifiScanner$ScanSettings;Landroid/net/wifi/WifiScanner$ScanListener;Landroid/os/WorkSource;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->startTrackingBssids([Landroid/net/wifi/WifiScanner$BssidInfo;ILandroid/net/wifi/WifiScanner$BssidListener;)V,sdk,system-api,test-api
@@ -2457,7 +2440,6 @@ Landroid/net/wifi/WifiScanner;->stopScan(Landroid/net/wifi/WifiScanner$ScanListe
 Landroid/net/wifi/WifiScanner;->stopTrackingBssids(Landroid/net/wifi/WifiScanner$BssidListener;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->stopTrackingWifiChange(Landroid/net/wifi/WifiScanner$WifiChangeListener;)V,sdk,system-api,test-api
 Landroid/net/wifi/WifiScanner;->unregisterScanListener(Landroid/net/wifi/WifiScanner$ScanListener;)V,sdk,system-api,test-api
-Landroid/net/wifi/WifiScanner;->validateChannel()V,lo-prio,max-target-o
 Landroid/net/wifi/WifiSsid$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiSsid$1;->newArray(I)[Ljava/lang/Object;,public-api,sdk,system-api,test-api
 Landroid/net/wifi/WifiSsid;->CREATOR:Landroid/os/Parcelable$Creator;,public-api,sdk,system-api,test-api
diff --git a/current/hiddenapi/filtered-stub-flags.csv b/current/hiddenapi/filtered-stub-flags.csv
index 498a53f..2143ca2 100644
--- a/current/hiddenapi/filtered-stub-flags.csv
+++ b/current/hiddenapi/filtered-stub-flags.csv
@@ -144,6 +144,10 @@ Landroid/net/wifi/IWifiScanner$Default;->asBinder()Landroid/os/IBinder;,public-a
 Landroid/net/wifi/IWifiScanner$Stub$Proxy;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
 Landroid/net/wifi/IWifiScanner$Stub;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
 Landroid/net/wifi/IWifiScanner$Stub;->onTransact(ILandroid/os/Parcel;Landroid/os/Parcel;I)Z,public-api,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Default;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Stub$Proxy;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Stub;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
+Landroid/net/wifi/IWifiScannerListener$Stub;->onTransact(ILandroid/os/Parcel;Landroid/os/Parcel;I)Z,public-api,system-api,test-api
 Landroid/net/wifi/IWifiVerboseLoggingStatusChangedListener$Default;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
 Landroid/net/wifi/IWifiVerboseLoggingStatusChangedListener$Stub$Proxy;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
 Landroid/net/wifi/IWifiVerboseLoggingStatusChangedListener$Stub;->asBinder()Landroid/os/IBinder;,public-api,system-api,test-api
@@ -1644,6 +1648,7 @@ Landroid/net/wifi/WifiNetworkSuggestion;->isUntrusted()Z,public-api,system-api,t
 Landroid/net/wifi/WifiNetworkSuggestion;->isUserInteractionRequired()Z,public-api,system-api,test-api
 Landroid/net/wifi/WifiNetworkSuggestion;->toString()Ljava/lang/String;,core-platform-api,public-api,system-api,test-api
 Landroid/net/wifi/WifiNetworkSuggestion;->writeToParcel(Landroid/os/Parcel;I)V,public-api,system-api,test-api
+Landroid/net/wifi/WifiScanner$$ExternalSyntheticLambda0;->run()V,core-platform-api,public-api,system-api,test-api
 Landroid/net/wifi/WifiScanner$ActionListener;->onFailure(ILjava/lang/String;)V,system-api,test-api
 Landroid/net/wifi/WifiScanner$ActionListener;->onSuccess()V,system-api,test-api
 Landroid/net/wifi/WifiScanner$BssidInfo;-><init>()V,system-api,test-api
@@ -1662,10 +1667,6 @@ Landroid/net/wifi/WifiScanner$HotlistSettings;->apLostThreshold:I,system-api,tes
 Landroid/net/wifi/WifiScanner$HotlistSettings;->bssidInfos:[Landroid/net/wifi/WifiScanner$BssidInfo;,system-api,test-api
 Landroid/net/wifi/WifiScanner$HotlistSettings;->describeContents()I,public-api,system-api,test-api
 Landroid/net/wifi/WifiScanner$HotlistSettings;->writeToParcel(Landroid/os/Parcel;I)V,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult$1;->newArray(I)[Ljava/lang/Object;,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult;->describeContents()I,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$OperationResult;->writeToParcel(Landroid/os/Parcel;I)V,public-api,system-api,test-api
 Landroid/net/wifi/WifiScanner$ParcelableScanData$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,system-api,test-api
 Landroid/net/wifi/WifiScanner$ParcelableScanData$1;->newArray(I)[Ljava/lang/Object;,public-api,system-api,test-api
 Landroid/net/wifi/WifiScanner$ParcelableScanData;-><init>([Landroid/net/wifi/WifiScanner$ScanData;)V,system-api,test-api
@@ -1722,12 +1723,11 @@ Landroid/net/wifi/WifiScanner$ScanSettings;->setRnrSetting(I)V,system-api,test-a
 Landroid/net/wifi/WifiScanner$ScanSettings;->stepCount:I,system-api,test-api
 Landroid/net/wifi/WifiScanner$ScanSettings;->type:I,system-api,test-api
 Landroid/net/wifi/WifiScanner$ScanSettings;->writeToParcel(Landroid/os/Parcel;I)V,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda0;->run()V,core-platform-api,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda1;->run()V,core-platform-api,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda2;->run()V,core-platform-api,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda3;->run()V,core-platform-api,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler$$ExternalSyntheticLambda4;->run()V,core-platform-api,public-api,system-api,test-api
-Landroid/net/wifi/WifiScanner$ServiceHandler;->handleMessage(Landroid/os/Message;)V,public-api,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda0;->run()V,core-platform-api,public-api,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda1;->run()V,core-platform-api,public-api,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda2;->run()V,core-platform-api,public-api,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda3;->run()V,core-platform-api,public-api,system-api,test-api
+Landroid/net/wifi/WifiScanner$ServiceListener$$ExternalSyntheticLambda4;->run()V,core-platform-api,public-api,system-api,test-api
 Landroid/net/wifi/WifiScanner$WifiChangeListener;->onChanging([Landroid/net/wifi/ScanResult;)V,system-api,test-api
 Landroid/net/wifi/WifiScanner$WifiChangeListener;->onQuiescence([Landroid/net/wifi/ScanResult;)V,system-api,test-api
 Landroid/net/wifi/WifiScanner$WifiChangeSettings$1;->createFromParcel(Landroid/os/Parcel;)Ljava/lang/Object;,public-api,system-api,test-api
diff --git a/current/hiddenapi/index.csv b/current/hiddenapi/index.csv
index b757cca..159c9e4 100644
--- a/current/hiddenapi/index.csv
+++ b/current/hiddenapi/index.csv
@@ -36,17 +36,17 @@ Landroid/net/wifi/WifiConfiguration;->defaultGwMacAddress:Ljava/lang/String;,pac
 Landroid/net/wifi/WifiConfiguration;->validatedInternetAccess:Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1347,5,1347,25,
 Landroid/net/wifi/WifiConfiguration;->INVALID_RSSI:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1431,5,1431,88,maxTargetSdk=28&trackingBug=115609023L
 Landroid/net/wifi/WifiConfiguration;->numNoInternetAccessReports:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1438,5,1438,88,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/WifiConfiguration;->noInternetAccessExpected:Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1458,5,1458,88,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/WifiConfiguration;->selfAdded:Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1512,5,1512,88,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/WifiConfiguration;->isEnterprise()Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3263,5,3263,25,
-Landroid/net/wifi/WifiConfiguration;->getStaticIpConfiguration()Landroid/net/StaticIpConfiguration;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3765,5,3765,25,
-Landroid/net/wifi/WifiConfiguration;->setStaticIpConfiguration(Landroid/net/StaticIpConfiguration;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3771,5,3771,25,
-Landroid/net/wifi/WifiConfiguration;->getIpAssignment()Landroid/net/IpConfiguration$IpAssignment;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3781,5,3781,25,
-Landroid/net/wifi/WifiConfiguration;->setIpAssignment(Landroid/net/IpConfiguration$IpAssignment;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3787,5,3787,25,
-Landroid/net/wifi/WifiConfiguration;->getProxySettings()Landroid/net/IpConfiguration$ProxySettings;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3797,5,3797,25,
-Landroid/net/wifi/WifiConfiguration;->setProxySettings(Landroid/net/IpConfiguration$ProxySettings;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3803,5,3803,25,
-Landroid/net/wifi/WifiConfiguration;->setProxy(Landroid/net/IpConfiguration$ProxySettings;Landroid/net/ProxyInfo;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3863,5,3863,25,
-Landroid/net/wifi/WifiConfiguration;->CREATOR:Landroid/os/Parcelable$Creator;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,4085,5,4085,25,
+Landroid/net/wifi/WifiConfiguration;->noInternetAccessExpected:Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1456,5,1456,88,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/WifiConfiguration;->selfAdded:Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,1510,5,1510,88,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/WifiConfiguration;->isEnterprise()Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3261,5,3261,25,
+Landroid/net/wifi/WifiConfiguration;->getStaticIpConfiguration()Landroid/net/StaticIpConfiguration;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3763,5,3763,25,
+Landroid/net/wifi/WifiConfiguration;->setStaticIpConfiguration(Landroid/net/StaticIpConfiguration;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3769,5,3769,25,
+Landroid/net/wifi/WifiConfiguration;->getIpAssignment()Landroid/net/IpConfiguration$IpAssignment;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3779,5,3779,25,
+Landroid/net/wifi/WifiConfiguration;->setIpAssignment(Landroid/net/IpConfiguration$IpAssignment;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3785,5,3785,25,
+Landroid/net/wifi/WifiConfiguration;->getProxySettings()Landroid/net/IpConfiguration$ProxySettings;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3795,5,3795,25,
+Landroid/net/wifi/WifiConfiguration;->setProxySettings(Landroid/net/IpConfiguration$ProxySettings;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3801,5,3801,25,
+Landroid/net/wifi/WifiConfiguration;->setProxy(Landroid/net/IpConfiguration$ProxySettings;Landroid/net/ProxyInfo;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,3861,5,3861,25,
+Landroid/net/wifi/WifiConfiguration;->CREATOR:Landroid/os/Parcelable$Creator;,packages/modules/Wifi/framework/java/android/net/wifi/WifiConfiguration.java,4083,5,4083,25,
 Landroid/net/wifi/WifiEnterpriseConfig;->mFields:Ljava/util/HashMap;,packages/modules/Wifi/framework/java/android/net/wifi/WifiEnterpriseConfig.java,250,5,250,25,
 Landroid/net/wifi/WifiEnterpriseConfig;->setCaCertificateAlias(Ljava/lang/String;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiEnterpriseConfig.java,718,5,718,25,
 Landroid/net/wifi/WifiEnterpriseConfig;->getCaCertificateAlias()Ljava/lang/String;,packages/modules/Wifi/framework/java/android/net/wifi/WifiEnterpriseConfig.java,769,5,769,25,
@@ -68,18 +68,18 @@ Landroid/net/wifi/WifiInfo;->setSupplicantState(Landroid/net/wifi/SupplicantStat
 Landroid/net/wifi/WifiInfo;->setSupplicantState(Ljava/lang/String;)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiInfo.java,1401,5,1401,25,
 Landroid/net/wifi/WifiInfo;->removeDoubleQuotes(Ljava/lang/String;)Ljava/lang/String;,packages/modules/Wifi/framework/java/android/net/wifi/WifiInfo.java,1430,5,1430,25,
 Landroid/net/wifi/WifiInfo;->CREATOR:Landroid/os/Parcelable$Creator;,packages/modules/Wifi/framework/java/android/net/wifi/WifiInfo.java,1571,5,1571,25,
-Landroid/net/wifi/WifiManager;->LINK_CONFIGURATION_CHANGED_ACTION:Ljava/lang/String;,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1265,5,1265,88,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/WifiManager;->MIN_RSSI:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1461,5,1461,25,
-Landroid/net/wifi/WifiManager;->MAX_RSSI:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1465,5,1465,25,
-Landroid/net/wifi/WifiManager;->RSSI_LEVELS:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1475,5,1475,88,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/WifiManager;->WIFI_FREQUENCY_BAND_AUTO:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1484,5,1484,25,
-Landroid/net/wifi/WifiManager;->WIFI_FREQUENCY_BAND_5GHZ:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1491,5,1491,25,
-Landroid/net/wifi/WifiManager;->WIFI_FREQUENCY_BAND_2GHZ:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1498,5,1498,25,
-Landroid/net/wifi/WifiManager;->mActiveLockCount:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1554,5,1554,25,
-Landroid/net/wifi/WifiManager;->mService:Landroid/net/wifi/IWifiManager;,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1558,5,1558,25,
-Landroid/net/wifi/WifiManager;->cancelLocalOnlyHotspotRequest()V,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,4942,5,4942,25,
-Landroid/net/wifi/WifiManager;->initializeMulticastFiltering()Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,6969,5,6969,25,
-Landroid/net/wifi/WifiManager;->enableVerboseLogging(I)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,7009,5,7012,6,maxTargetSdk=29&publicAlternatives=%22Use+%7B%40code+%23setVerboseLoggingEnabled%28boolean%29%7D+instead.%22
+Landroid/net/wifi/WifiManager;->LINK_CONFIGURATION_CHANGED_ACTION:Ljava/lang/String;,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1287,5,1287,88,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/WifiManager;->MIN_RSSI:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1483,5,1483,25,
+Landroid/net/wifi/WifiManager;->MAX_RSSI:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1487,5,1487,25,
+Landroid/net/wifi/WifiManager;->RSSI_LEVELS:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1497,5,1497,88,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/WifiManager;->WIFI_FREQUENCY_BAND_AUTO:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1506,5,1506,25,
+Landroid/net/wifi/WifiManager;->WIFI_FREQUENCY_BAND_5GHZ:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1513,5,1513,25,
+Landroid/net/wifi/WifiManager;->WIFI_FREQUENCY_BAND_2GHZ:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1520,5,1520,25,
+Landroid/net/wifi/WifiManager;->mActiveLockCount:I,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1576,5,1576,25,
+Landroid/net/wifi/WifiManager;->mService:Landroid/net/wifi/IWifiManager;,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,1580,5,1580,25,
+Landroid/net/wifi/WifiManager;->cancelLocalOnlyHotspotRequest()V,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,4964,5,4964,25,
+Landroid/net/wifi/WifiManager;->initializeMulticastFiltering()Z,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,7006,5,7006,25,
+Landroid/net/wifi/WifiManager;->enableVerboseLogging(I)V,packages/modules/Wifi/framework/java/android/net/wifi/WifiManager.java,7046,5,7046,6,maxTargetSdk=29&publicAlternatives=%22Use+%7B%40code+%23setVerboseLoggingEnabled%28boolean%29%7D+instead.%22
 Landroid/net/wifi/WifiSsid;->octets:Ljava/io/ByteArrayOutputStream;,packages/modules/Wifi/framework/java/android/net/wifi/WifiSsid.java,206,5,206,69,publicAlternatives=%22%7B%40link+%23getBytes%28%29%7D%22
 Landroid/net/wifi/WifiSsid;->NONE:Ljava/lang/String;,packages/modules/Wifi/framework/java/android/net/wifi/WifiSsid.java,214,5,214,99,publicAlternatives=%22%7B%40link+android.net.wifi.WifiManager%23UNKNOWN_SSID%7D%22
 Landroid/net/wifi/WifiSsid;->createFromAsciiEncoded(Ljava/lang/String;)Landroid/net/wifi/WifiSsid;,packages/modules/Wifi/framework/java/android/net/wifi/WifiSsid.java,222,5,222,76,publicAlternatives=%22%7B%40link+%23fromBytes%28byte%5B%5D%29%7D%22
@@ -101,12 +101,12 @@ Landroid/net/wifi/p2p/WifiP2pGroup;->setInterface(Ljava/lang/String;)V,packages/
 Landroid/net/wifi/p2p/WifiP2pGroup;->setNetworkId(I)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pGroup.java,285,5,285,25,
 Landroid/net/wifi/p2p/WifiP2pGroupList;->mGroups:Landroid/util/LruCache;,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pGroupList.java,42,5,42,25,
 Landroid/net/wifi/p2p/WifiP2pGroupList;-><init>(Landroid/net/wifi/p2p/WifiP2pGroupList;Landroid/net/wifi/p2p/WifiP2pGroupList$GroupDeleteListener;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pGroupList.java,59,5,59,25,
-Landroid/net/wifi/p2p/WifiP2pManager$Channel;->mAsyncChannel:Lcom/android/internal/util/AsyncChannel;,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,1225,9,1225,29,
-Landroid/net/wifi/p2p/WifiP2pManager$Channel;->putListener(Ljava/lang/Object;)I,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,1469,9,1469,92,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/p2p/WifiP2pManager;->CREATE_GROUP:I,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,544,5,544,88,maxTargetSdk=30&trackingBug=170729553L
-Landroid/net/wifi/p2p/WifiP2pManager;-><init>(Landroid/net/wifi/p2p/IWifiP2pManager;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,785,5,785,88,maxTargetSdk=28&trackingBug=115609023L
-Landroid/net/wifi/p2p/WifiP2pManager;->startWps(Landroid/net/wifi/p2p/WifiP2pManager$Channel;Landroid/net/wifi/WpsInfo;Landroid/net/wifi/p2p/WifiP2pManager$ActionListener;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,2000,5,2000,50,trackingBug=185141982L
-Landroid/net/wifi/p2p/WifiP2pManager;->setWFDInfo(Landroid/net/wifi/p2p/WifiP2pManager$Channel;Landroid/net/wifi/p2p/WifiP2pWfdInfo;Landroid/net/wifi/p2p/WifiP2pManager$ActionListener;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,2340,5,2340,88,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/p2p/WifiP2pManager$Channel;->mAsyncChannel:Lcom/android/internal/util/AsyncChannel;,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,1232,9,1232,29,
+Landroid/net/wifi/p2p/WifiP2pManager$Channel;->putListener(Ljava/lang/Object;)I,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,1476,9,1476,92,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/p2p/WifiP2pManager;->CREATE_GROUP:I,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,551,5,551,88,maxTargetSdk=30&trackingBug=170729553L
+Landroid/net/wifi/p2p/WifiP2pManager;-><init>(Landroid/net/wifi/p2p/IWifiP2pManager;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,792,5,792,88,maxTargetSdk=28&trackingBug=115609023L
+Landroid/net/wifi/p2p/WifiP2pManager;->startWps(Landroid/net/wifi/p2p/WifiP2pManager$Channel;Landroid/net/wifi/WpsInfo;Landroid/net/wifi/p2p/WifiP2pManager$ActionListener;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,2027,5,2027,50,trackingBug=185141982L
+Landroid/net/wifi/p2p/WifiP2pManager;->setWFDInfo(Landroid/net/wifi/p2p/WifiP2pManager$Channel;Landroid/net/wifi/p2p/WifiP2pWfdInfo;Landroid/net/wifi/p2p/WifiP2pManager$ActionListener;)V,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pManager.java,2368,5,2368,88,maxTargetSdk=30&trackingBug=170729553L
 Landroid/net/wifi/p2p/WifiP2pProvDiscEvent;->event:I,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pProvDiscEvent.java,37,5,37,25,
 Landroid/net/wifi/p2p/WifiP2pProvDiscEvent;->device:Landroid/net/wifi/p2p/WifiP2pDevice;,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pProvDiscEvent.java,40,5,40,25,
 Landroid/net/wifi/p2p/WifiP2pProvDiscEvent;->pin:Ljava/lang/String;,packages/modules/Wifi/framework/java/android/net/wifi/p2p/WifiP2pProvDiscEvent.java,44,5,44,25,
diff --git a/current/hiddenapi/metadata.csv b/current/hiddenapi/metadata.csv
index 63c5e5e..839193c 100644
--- a/current/hiddenapi/metadata.csv
+++ b/current/hiddenapi/metadata.csv
@@ -2,6 +2,9 @@ signature,maxTargetSdk,overrideSourcePosition,publicAlternatives,trackingBug
 |Landroid/net/wifi/IWifiManager;->getCurrentNetwork()Landroid/net/Network;|,|30|,|packages/modules/Wifi/framework/java/android/net/wifi/IWifiManager.aidl:255:1:255:69|,||,|170729553|
 |Landroid/net/wifi/IWifiManager;->getWifiApConfiguration()Landroid/net/wifi/WifiConfiguration;|,||,|packages/modules/Wifi/framework/java/android/net/wifi/IWifiManager.aidl:230:1:230:25|,||,||
 |Landroid/net/wifi/IWifiManager;->getWifiApEnabledState()I|,||,|packages/modules/Wifi/framework/java/android/net/wifi/IWifiManager.aidl:227:1:227:25|,||,||
+|Landroid/net/wifi/IWifiManager;->getCurrentNetwork()Landroid/net/Network;|,|30|,|packages/modules/Wifi/framework/java/android/net/wifi/IWifiManager.aidl:255:1:255:69|,||,|170729553|
+|Landroid/net/wifi/IWifiManager;->getWifiApConfiguration()Landroid/net/wifi/WifiConfiguration;|,||,|packages/modules/Wifi/framework/java/android/net/wifi/IWifiManager.aidl:230:1:230:25|,||,||
+|Landroid/net/wifi/IWifiManager;->getWifiApEnabledState()I|,||,|packages/modules/Wifi/framework/java/android/net/wifi/IWifiManager.aidl:227:1:227:25|,||,||
 |Landroid/net/wifi/ScanResult$InformationElement;->EID_BSS_LOAD:I|,|30|,||,||,|170729553|
 |Landroid/net/wifi/ScanResult$InformationElement;->EID_ERP:I|,|30|,||,||,|170729553|
 |Landroid/net/wifi/ScanResult$InformationElement;->EID_EXTENDED_CAPS:I|,|30|,||,||,|170729553|
diff --git a/current/hiddenapi/signature-patterns.csv b/current/hiddenapi/signature-patterns.csv
index 33c2b4b..ef996fa 100644
--- a/current/hiddenapi/signature-patterns.csv
+++ b/current/hiddenapi/signature-patterns.csv
@@ -28,6 +28,7 @@ android/net/wifi/ITrafficStateCallback
 android/net/wifi/IWifiConnectedNetworkScorer
 android/net/wifi/IWifiManager
 android/net/wifi/IWifiScanner
+android/net/wifi/IWifiScannerListener
 android/net/wifi/IWifiVerboseLoggingStatusChangedListener
 android/net/wifi/MloLink
 android/net/wifi/ParcelUtil
diff --git a/current/snapshot-creation-build-number.txt b/current/snapshot-creation-build-number.txt
index d710cd0..e1a9fb8 100644
--- a/current/snapshot-creation-build-number.txt
+++ b/current/snapshot-creation-build-number.txt
@@ -1 +1 @@
-9279849
\ No newline at end of file
+9832159
-- 
2.17.1

